use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,358 @@
+#Introduction
+
+Intel\* QuickAssist Technology (Intel\* QAT) provides hardware acceleration of security (cryptography) and compression. These instructions cover the steps for Clear Linux\* but can be adapted to any Linux\* distribution. Your distribution may already have the Intel QAT drivers, but it is likely they are out of date and do not contain the necessary user space components.  These instructions guide the user on how to download the kernel sources, compile kernel driver modules against those sources, and load them onto the host as well as preparing a specially built Kata Containers® kernel and custom Kata Containers rootfs.  
+
+##Helpful Links before starting
+[https://github.com/intel/QAT_Engine](https://github.com/intel/QAT_Engine)
```

Comment:
Please make this into a set of bullets with descriptive names for the links rather than just the link address, such as:

> - [Intel QAT engine project](https://github.com/intel/QAT_Engine)
>
> - [Intel QuickAssist Technology project](https://01.org/intel-quickassist-technology)